### PR TITLE
Adding new term GitOps

### PR DESCRIPTION
**Pull request by jgershater**

### CODE REVIEW

**Comment by caniszczyk** [comment]
We should probably reference what the GitOps WG put together https://github.com/open-gitops/documents/blob/main/GLOSSARY.md

**Comment by caniszczyk** [comment]
in particular, this definition https://github.com/open-gitops/documents/blob/main/PRINCIPLES.md#gitops-principles-version

**Comment by caniszczyk** [comment]
LGTM

**Review by caniszczyk — APPROVED** [review]

**Comment by jgershater** [comment]
I made a few edits to the cloud-native computing entry

**Review by jgershater — COMMENTED** [review]
I made a few edits to the cloud-native application entry

**Comment by JasonMorgan** [comment]
@jgershater we need to restrict this PR to changes on a single entry. Please remove the changes to cloud native apps.

**Review by JasonMorgan — CHANGES_REQUESTED** [review]
Need to restrict this PR to a single entry.

**Comment by CathPag** [comment]
Thanks, @jgershater. I made a few suggestions below for the GitOps entry. I simplified the what it is section. For instance, "describes the desired state of the entire system" is not necessarily something someone without a technical background would understand. 

The problem it addresses needs to talk about the problem. Right now it says how it helps administrators and that goes into the how it helps section.

Also, I see you created this as an .md file but it's not popping up as such (not sure why). And, as Jason pointed out, we need to split this PR in two. The cloud native apps update should be its own PR.

Let me know if you have any questions. 

## What it is
GitOps is a [set of principles](https://github.com/open-gitops/documents/blob/main/PRINCIPLES.md) for operating and managing software systems. In GitOps, environment specifics are defined, or "declared," in a repository with the application code — your so-called system of record. 

## Problem it addresses
[here we need to describe the problems that exist when you don't capture environment specifics in a git.]

## How it helps
A GitOps approach provides a complete history of how environments have been modified over time. That allows operators to revert to a previous state if needed, significantly easing error recovery.

**Comment by hseligson1** [comment]
Just some suggestions for the sections @CathPag mentions:

**Problems it addresses:**
The GitOps paradigm addresses problems organizations face such as decreased developer productivity and experience for teams, frequently failed deployments that rely on long and slow disaster recovery from system outages due to unplanned downtime, and a lack of visibility of your system's workflow, among many other challenges.

**How it helps:**
By adopting GitOps within an application or system workflow an organization can increase their team's productivity, deploy more frequently, automate rollbacks and remediate outages quickly and effectively by leveraging visibility thanks to their Git history within the source control system.

Please let me know if this is high-level enough!

> Thanks, @jgershater. I made a few suggestions below for the GitOps entry. I simplified the what it is section. For instance, "describes the desired state of the entire system" is not necessarily something someone without a technical background would understand.
> 
> The problem it addresses needs to talk about the problem. Right now it says how it helps administrators and that goes into the how it helps section.
> 
> Also, I see you created this as an .md file but it's not popping up as such (not sure why). And, as Jason pointed out, we need to split this PR in two. The cloud native apps update should be its own PR.
> 
> Let me know if you have any questions.
> 
> ## What it is
> GitOps is a [set of principles](https://github.com/open-gitops/documents/blob/main/PRINCIPLES.md) for operating and managing software systems. In GitOps, environment specifics are defined, or "declared," in a repository with the application code — your so-called system of record.
> 
> ## Problem it addresses
> [here we need to describe the problems that exist when you don't capture environment specifics in a git.]
> 
> ## How it helps
> A GitOps approach provides a complete history of how environments have been modified over time. That allows operators to revert to a previous state if needed, significantly easing error recovery.

**Comment by CathPag** [comment]
Looks great, @hseligson1 — thank you! @jgershater can you incorporate the feedback? We're getting very close 🎉

**Comment by CathPag** [comment]
@jgershater, I incorporated some of @hseligson1's suggestions and I think it's now a mix of both of your input. What do you and @hseligson1 think?

**Comment by hseligson1** [comment]
> 

I agree @CathPag - @jgershater content is a nice mixture! I'd move forward with this.

File: GitOps.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+
+## What it is
+
+GitOps is a [set of principles](https://github.com/open-gitops/documents/blob/main/PRINCIPLES.md) for operating and managing software systems. In GitOps, environment specifics are defined, or "declared," in a repository with the application code — the system of record.
+
+## Problem it addresses
+
+Configuration management via YAML or JSON files often becomes unwieldy at scale. These files can grow exponentially, especially in large organizations with numerous developers and frequent code changes. Potential consequences include frequently failed deployments (e.g., usage of the incorrect config files), long and slow disaster recovery from system outages, lack of visibility into system workflows, reduced developer productivity, among other challenges.
```
Comment (JasonMorgan) [inline GitOps.md:12]: I think we've missed something here. GitOps doesn't solve for the problem of storing config in JSON or YAML. I agree that GitOps solves for, or at least improves DR, but many of the other problems and their root cause suggested in this section aren't addressed by a GitOps workflow.

File: GitOps.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: GitOps
+status: Feedback Appreciated
+category: concept
+---
+
+
+## What it is
+
+GitOps is a [set of principles](https://github.com/open-gitops/documents/blob/main/PRINCIPLES.md) for operating and managing software systems. In GitOps, environment specifics are defined, or "declared," in a repository with the application code — the system of record.
```
Comment (JasonMorgan) [inline GitOps.md:9]: Good but we need a few more lines to describe what it is beyond principles. GitOps is also a set of practices that involve applying that stored state in a consistent way. I appreciate that we're attempting to weave in the work produced by the gitops working group but their definition is tailored to an audience of existing gitops practitioners. Our definition needs to be more expansive.

**Review by JasonMorgan — CHANGES_REQUESTED** [review]

**Comment by JasonMorgan** [comment]
Closed due to inactivity. @hseligson1 please feel free to open a new PR for this definition.

**Comment by hseligson1** [comment]
> 

Will do - I should be able to submit a PR today.

**Pull request closed without merging**